generator/src/test/java/com/linkedin/pegasus/generator/TestTemplateSpecGenerator.java

```diff
@@ -72,14 +74,18 @@
   private AtomicInteger _uniqueNumberGenerator;
   @Mock
   private DataSchemaResolver _resolver;
-  @Mock
   private DataSchemaLocation _location;
 
   @BeforeMethod
   public void initMocks() {
     MockitoAnnotations.initMocks(this);
     _uniqueNumberGenerator = new AtomicInteger();
-    Mockito.when(_resolver.nameToDataSchemaLocations()).thenReturn(Collections.singletonMap(INPUT_SCHEMA_NAME, _location));
+    _location = new StringDataSchemaLocation("location");
+    HashMap<String, DataSchemaLocation> schemaToLocation = new HashMap<>();
+    schemaToLocation.put(INPUT_SCHEMA_NAME, _location);
+    schemaToLocation.put(CUSTOM_TYPE_1.getFullName(), _location);
+    schemaToLocation.put(CUSTOM_TYPE_2.getFullName(), _location);
+    Mockito.when(_resolver.nameToDataSchemaLocations()).thenReturn(schemaToLocation);
```

Comment:
This change here is happening is only because the test wasn't need to mock the schema location before, now it has to, so I have to patch this test, it is not because of any additional logic change